_includes/master/charts/calico/templates/calico-typha.yaml

```diff
@@ -111,6 +111,7 @@ spec:
         securityContext:
           runAsNonRoot: true
           allowPrivilegeEscalation: false
+          fsGroup: 65534
```

Comment:
This is in the wrong spot - it needs to be in the pod's security context, not the container's.

Reply:
Also, could you add a comment linking to this issue which explains why we need this? https://github.com/kubernetes/kubernetes/issues/82573